src/components/Login/utils.test.tsx

```diff
@@ -20,9 +20,13 @@ describe("decodeQr", () => {
   it("should throw if the code can be parsed but does not contain the right fields", () => {
     expect.assertions(2);
     const missingKey = `{"keys": "1e4457bc-f7d0-4329-a344-f0e3c75d8dd4","endpoint": "https://somewhere.com"}`;
-    expect(() => decodeQr(missingKey)).toThrow("No key specified");
+    expect(() => decodeQr(missingKey)).toThrow(
+      "We are currently facing login issues. Get a new QR code from your in-charge."
```

Comment:
is it better to use `ERROR_MESSAGE.AUTH_FAILURE_INVALID_QR` here?

Reply:
hmm, for test cases now it seems we dont use the `ERROR_MESSAGE` enums from alert context, should we leave this as the full string just for the unit tests?

Reply:
sure!